### DISCUSSION TITLE
Theme of ggplotly

### DISCUSSION THREAD

**Discussion by DeanAlexander27** [teamalgoritma/community#575] [Q&A] [answered]
Dear Ka Instruktur

apakah ada referensi lain mengenai tema pada ggplotly ?
saya tertarik dengan tema yang dipakai referensi pada DV capstone seperti ini
mungkin ada referensi yang dapat diberikan
![image](https://user-images.githubusercontent.com/123348110/221332648-af3486bb-0c75-4e94-b3d8-c8f0315e00d5.png)

terima kasih sehat selalu

**Comment by idoali** [comment, marked as answer]
Dear Pak Dean

Terima kasih atas pertanyaannya pak. Fungsi ggplotly adalah fungsi yang digunakan untuk mengubah plot ggplot menjadi plot plotly. Maka apa yang akan ditampilkan di dashboard sebenarnya adalah plot plotly hasil terjemahan ggplot. Maka dari itu, untuk mengatur temanya bisa mengaturnya langsung pada `theme()` ggplot. 

Yang ditampilkan pada dashboard diatas sendiri bukan `ggplotly` pak, melainkan `echarts4r`, library yang lain lagi untuk membuat visualisasi. 

Jika bapak ingin tahu lebih jauh mengenai library `echarts4r`, bapak bisa buka dokumentasi berikut. Disana juga terdapat video tutorial yang mungkin bisa membantu. 

https://echarts4r.john-coene.com/
